ensime_shared/util.py

```diff
@@ -21,6 +21,22 @@ def mkdir_p(path):
         if not os.path.exists(path):
             os.makedirs(path)
 
+    @staticmethod
+    def extract_packageName(lines):
+        foundPacakge = False
```

Comment:
Minor style issue: don't use camel case in python, use underscores to separate names.